include/sensor_fusion.h

```diff
@@ -0,0 +1,103 @@
+/**
+ * \brief Function Prototype Library to implement Sensor Fusion Algorithm
+ * \author jaspalsingh@cmail.carleton.ca
+ * \version 1.0
+ * \date 2019-12-01
+ */
+
+#ifndef SENSOR_FUSION_H
+#define SENSOR_FUSION_H
+
+/**
+ * \brief Structure for storing support degree matrix
+ * \details For the given number of sensors the structure will store support degree matrix
+ */
+struct support_degree_matrix {
+    int sensor_count;
+    double *sd_matrix;
+};
```

Comment:
Structures should include a typedef so that you don't need to include the struct keyword throughout the code. You can read more about this on slide 10 of the coding conventions presentation. For example, this struct would become:
```
typedef struct support_degree_matrix {
    int sensor_count;
    double *sd_matrix;
} support_degree_matrix_t;
```

Reply:
Fixed in recent commits to dev branch